### PR TITLE
Update notation accessibility information when notation is edited

### PR DESCRIPTION
**Pull request by cbjeukendrup**
Resolves: #10337

### CODE REVIEW

File: src/notation/internal/notationaccessibility.h [hunk 1]
```diff
@@ -40,9 +40,9 @@ class IGetScore;
 class NotationAccessibility : public INotationAccessibility, public async::Asyncable
 {
 public:
-    NotationAccessibility(const IGetScore* getScore, async::Notification selectionChangedNotification);
+    NotationAccessibility(const Notation* notation);
 
-    ValCh<std::string> accessibilityInfo() const override;
```
Comment (Eism) [inline src/notation/internal/notationaccessibility.h:45]: It's better to keep std::string here

Comment (Eism) [reply src/notation/internal/notationaccessibility.h:45]: > in the future we will rewrite libmscore to use std::string too?

yes

**Review by Eism — APPROVED** [review]

**Comment by DmitryArefiev** [comment]
Tested #10337 on Win10, Mac12, LinuxMint 20.2 - FIXED      

Please merge into master

**Comment by DmitryArefiev** [comment]
@Eism But screenreader doesn't announce the changes when moving notes up/down. Will log a separate bug for that

**Pull request merged**